src/components/student-dashboard/empty-panel.cjsx

```diff
@@ -0,0 +1,12 @@
+React = require 'react'
+BS = require 'react-bootstrap'
+
+module.exports = React.createClass
+
+  displayName: 'EmptyPanel'
+
+  render: ->
+    content = @props.children or "No events this week"
+    <BS.Panel header={@props.title}>
```

Comment:
can you add `className="empty"` to the panel instead of the inner div? That way the whole panel could be hidden.